### PR TITLE
Quantize and Dequantize documentation

### PR DESCRIPTION
**Pull request by adstraw**

### CODE REVIEW

File: doc/sphinx/source/ops/dequantize.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+   Dequantize // Maps quantized input to real output using scale and offset
+
+Description
+===========
+
+Produces a tensor of element type ``type`` and the same shape as ``input``
+where the value of each coordinate (i) of ``output`` is the corresponding coordinate of 
+``input`` plus ``offset`` quantity multiplied by ``scale``.  
+The coordinate (j) of ``scale`` and ``offset`` is the coordinate of ``output``
+projected along ``axes``.
```
Comment (aprocter) [inline doc/sphinx/source/ops/dequantize.rst]: Should we say `along` or `onto`?

File: doc/sphinx/source/ops/dequantize.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Inputs
+------
+
++-----------------+-------------------------+---------------------------------------+
+| Name            | Element Type            | Shape                                 |
++=================+=========================+=======================================+
+| ``input``       | is_quantized()          | Any                                   |
```
Comment (aprocter) [inline doc/sphinx/source/ops/dequantize.rst]: Not sure if this is adhering to an existing convention, but if not I'd prefer something like "Any quantized type" instead of `is_quantized()`.

File: doc/sphinx/source/ops/quantize.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+| ``offset``      | Same as ``output``      | ``input`` shape projected on ``axes`` |
++-----------------+-------------------------+---------------------------------------+
+
+Attributes
+----------
+
++-------------------------------+----------------------------------------------------------------+
+| Name                          | Description                                                    |
++===============================+================================================================+
+| ``type``                      | ``output`` element type                                        |
```
Comment (aprocter) [inline doc/sphinx/source/ops/quantize.rst]: Would rephrase as "The output element, type, which must be a quantized type", and see below...

File: doc/sphinx/source/ops/quantize.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+| ``round_mode``                | See src/ngraph/op/quantize.hpp                                 |
++-------------------------------+----------------------------------------------------------------+
+
+Outputs
+-------
+
++-----------------+-------------------------+---------------------------------------+
+| Name            | Element Type            | Shape                                 |
++=================+=========================+=======================================+
+| ``output``      | is_quantized()          | Same as ``input``                     |
```
Comment (aprocter) [inline doc/sphinx/source/ops/quantize.rst]: ...would change `is_quantized` to `type`.

File: doc/sphinx/source/ops/quantize.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+| ``offset``      | Same as ``output``      | ``input`` shape projected on ``axes`` |
++-----------------+-------------------------+---------------------------------------+
+
+Attributes
+----------
+
++-------------------------------+----------------------------------------------------------------+
+| Name                          | Description                                                    |
++===============================+================================================================+
+| ``type``                      | The output element, type, which must be a quantized type       |
```
Comment (aprocter) [inline doc/sphinx/source/ops/quantize.rst]: There's a stray comma between `element` and `type` here now.

File: doc/sphinx/source/ops/dequantize.rst [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+.. code-block:: cpp
+
+   Dequantize // Maps quantized input to real output using scale and offset
+
+Description
+===========
+
+Produces a tensor of element type ``type`` and the same shape as ``input``
+where the value of each coordinate (i) of ``output`` is the corresponding coordinate of
```
Comment (indie) [inline doc/sphinx/source/ops/dequantize.rst]: :math:`i`

File: doc/sphinx/source/ops/dequantize.rst [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+   Dequantize // Maps quantized input to real output using scale and offset
+
+Description
+===========
+
+Produces a tensor of element type ``type`` and the same shape as ``input``
+where the value of each coordinate (i) of ``output`` is the corresponding coordinate of 
+``input`` plus ``offset`` quantity multiplied by ``scale``.  
+The coordinate (j) of ``scale`` and ``offset`` is the coordinate of ``output``
```
Comment (indie) [inline doc/sphinx/source/ops/dequantize.rst]: :math:`j`
